frontend-react/src/pages/submissions/SubmissionPages.css

```diff
@@ -0,0 +1,16 @@
+
+.filter-container {
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 1em;
+}
+
+.apply-button {
+    background: none;
+    border: none;
+}
+
+.date-picker-container {
+    max-width: 225px;
+    margin-right: 32px;
+}
```

Comment:
Humm... should `.css` files go into the style folder or should they go along side the file that uses?

Probably a bigger team decision to be consistent. (I've been adding sections to the existing .scss main file.)

Keep in mind that these individual files get transpiled into a single .css file and many of the "static site" libraries (Gatsby) have a namespace feature that keeps them from colliding (e.g. another `.apply-button` or `.filter-container` could exist in another .css or .scss file). Which is an argument to put them all in one file to make it easier to spot?

Reply:
My only concern with a single file is that the application grows pretty fast and I'm not sure how many custom styles we'll need, so it's hard to know whether that file will become unruly or not. Perhaps a naming convention could help us keep things organized even if we do the multiple file approach? Additionally, would scss throw an error in that instance or help us in any way? 🤔 If that's something that can halt a build, then we're covered, but if I recall, it's not.